crates/jstz_cli/src/repl/debug_api/account.rs

```diff
@@ -0,0 +1,133 @@
+use std::ops::Deref;
```

Comment:
I guess this was just moved from previous debug_api in the jstz_cli - not sure why Github shows it as new files. 

Not sure if I am very supportive of this movement - I think it would be better to rather have all APIs in a single place - e.g. jstz_api.  In this way it is quite hidden and I think especially if someone new joins jstz development in the future it may be easier to intuitively understand the code if all APIs can be seen together. We may also decide at some point to enable debugging with DebugAPI in the actual .js contracts.

Reply:
Ah I see now why it is not showing as movement - there are actually two debug_api folders now. Please remove one of them.

Reply:
Likely an artefact from moving the folder. Will likely disappear once the PR is split

Reply:
Resolving since diff nolonger exists